tools/export/uvision4.py

```diff
@@ -28,7 +28,7 @@ class Uvision4(Exporter):
     Exporter class for uvision. This class uses project generator.
     """
     # These 2 are currently for exporters backward compatiblity
-    NAME = 'uVision4'
+    NAME = 'uvision'
```

Comment:
what targets use uv4 that dont use progen. This may be a good chance to push all uv support there